CanonicalLayouts/list-detail-compose/app/src/main/java/com/example/listdetailcompose/ui/ListDetailSample.kt

```diff
@@ -56,59 +74,84 @@ private val loremIpsum = """
         |Tempus quam pellentesque nec nam aliquam. Praesent semper feugiat nibh sed. Adipiscing elit duis tristique sollicitudin nibh sit. Netus et malesuada fames ac turpis egestas sed tempus urna. Quis varius quam quisque id diam vel quam. Urna duis convallis convallis tellus id interdum velit laoreet. Id eu nisl nunc mi ipsum. Fermentum dui faucibus in ornare. Nunc lobortis mattis aliquam faucibus. Vulputate mi sit amet mauris commodo quis. Porta nibh venenatis cras sed. Vitae tortor condimentum lacinia quis vel eros donec. Eu non diam phasellus vestibulum.
         """.trimMargin()
 private val sampleWords = listOf(
-    "Apple" to loremIpsum,
-    "Banana" to loremIpsum,
-    "Cherry" to loremIpsum,
-    "Date" to loremIpsum,
-    "Elderberry" to loremIpsum,
-    "Fig" to loremIpsum,
-    "Grape" to loremIpsum,
-    "Honeydew" to loremIpsum,
-).map { (word, definition) -> DefinedWord(word, definition) }
+    "Apple" to R.drawable.ic_food,
+    "Banana" to R.drawable.ic_no_food,
+    "Cherry" to R.drawable.ic_food,
+    "Date" to R.drawable.ic_no_food,
+    "Elderberry" to R.drawable.ic_food,
+    "Fig" to R.drawable.ic_no_food,
+    "Grape" to R.drawable.ic_food,
+    "Honeydew" to R.drawable.ic_no_food,
+).map { (word, icon) -> DefinedWord(word, icon) }
 
 private data class DefinedWord(
     val word: String,
-    val definition: String
+    @DrawableRes val icon: Int,
+    val definition: String = loremIpsum
 )
 
+@SuppressLint("UnusedContentLambdaTargetStateParameter")
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
 fun ListDetailSample() {
     var selectedWordIndex: Int? by rememberSaveable { mutableStateOf(null) }
     val navigator = rememberListDetailPaneScaffoldNavigator<Nothing>()
+    val isListAndDetailVisible =
+        currentWindowAdaptiveInfo().windowSizeClass.windowWidthSizeClass == WindowWidthSizeClass.EXPANDED
```

Comment:
And then the last thing I was thinking was to actually make this be directly calculate if both are visible with
```suggestion
        navigator.scaffoldValue[ListDetailPaneScaffoldRole.Detail] == PaneAdaptedValue.Expanded && navigator.scaffoldValue[ListDetailPaneScaffoldRole.List] == PaneAdaptedValue.Expanded
```

Instead of inferring the same thing from the window size class.